src/resolve.jl

```diff
@@ -490,27 +497,28 @@ function resolve(; force::Bool=false, io::IO=stderr, interactive::Bool=false, dr
                 changed = true
                 _resolve_pip_install(io, pip_specs, load_path)
             end
-            if !changed
-                _log(io, "Dependencies already up to date")
-            end
+            changed || _log(io, "Dependencies already up to date")
         else
             # the state is too dirty, recreate the conda environment from scratch
             dry_run && return
             # remove environment
             mkpath(meta_dir)
-            if isdir(conda_env)
+            create = true
+            isdir(conda_env) && if shared
```

Comment:
Combining `&&` and `if` in a single expression is kinda confusing - please just make it two `if`s.

Reply:
As you wish.